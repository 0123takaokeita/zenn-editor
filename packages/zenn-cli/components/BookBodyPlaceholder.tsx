export const BookBodyPlaceholder: React.FC = () => (
  <>
    <h1>✍️ チャプターを作成する</h1>
    <p>
      1つめのチャプターを作成しましょう。チャプターは
      <code>チャプター番号.md</code>
      という形式でファイルを作成します。チャプター1は
      <code>1.md</code>、チャプター2は
      <code>2.md</code>…のようになります。
      <br />
      <br />
      <a
        href="https://zenn.dev/zenn/how-to-use-zenn-cli"
        target="_blank"
        rel="noreferrer noopener"
      >
        本の作成について詳しく知る→
      </a>
    </p>
  </>
);
