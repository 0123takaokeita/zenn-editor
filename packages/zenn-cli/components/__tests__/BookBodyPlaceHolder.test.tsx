import React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { BookBodyPlaceholder } from '@components/BookBodyPlaceholder';

const renderer = createRenderer();
describe('<BookBodyPlaceHolder />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<BookBodyPlaceholder />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
