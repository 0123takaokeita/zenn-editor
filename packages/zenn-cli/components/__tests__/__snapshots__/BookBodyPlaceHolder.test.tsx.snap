// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BookBodyPlaceHolder /> should render and match the snapshot 1`] = `
<React.Fragment>
  <h1>
    📝 チャプターを作成する
  </h1>
  <p>
    1つめのチャプターを作成しましょう。チャプターは
    <code>
      チャプター番号.md
    </code>
    という形式でファイルを作成します。チャプター1は
    <code>
      1.md
    </code>
    、チャプター2は
    <code>
      2.md
    </code>
    …のようになります。
    <br />
    <br />
    <a
      href="https://zenn.dev/zenn/articles/zenn-cli-guide#cliで本（book）を管理する"
      rel="noreferrer noopener"
      target="_blank"
    >
      本の作成について詳しく知る→
    </a>
  </p>
</React.Fragment>
`;
